platform/ui/src/utils/SnackbarProvider.js

```diff
@@ -25,6 +25,11 @@ const SnackbarProvider = ({ children, service }) => {
   const [count, setCount] = useState(1);
   const [snackbarItems, setSnackbarItems] = useState([]);
 
+  /**
+   * Sets the implementation of a notification service that can be used by extensions.
+   *
+   * @returns void
+   */
```

Comment:
Oops. We should probably make sure that if `service` is optional that we don't call `service.setServiceImplementation({ hide, show });` if it's not set

Reply:
Done! thank you.